lib/store.js

```diff
@@ -1,18 +1,31 @@
 var Promise = require('bluebird');
 var Collection = require('./collection');
-var Store = function(mongodb) {
+var MongoOplog = require('mongo-oplog');
+
+var Store = function(mongodb, oplogEnabled) {
 	this._mongodb = mongodb;
 	this._collections = {};
-}
+	this._oplogEnabled = oplogEnabled;
+	if (oplogEnabled) {
```

Comment:
Maybe a bit unnecessary to start the oplog before we have any observers, but I guess in reality it should be fine?